src/librustc_mir/borrow_check/nll/region_infer/error_reporting/region_name.rs

```diff
@@ -279,11 +284,30 @@ impl<'tcx> RegionInferenceContext<'tcx> {
 
             ty::ReFree(free_region) => match free_region.bound_region {
                 ty::BoundRegion::BrNamed(_, name) => {
+                    // Get the span to point to, even if we don't use the name.
                     let span = self.get_named_span(tcx, error_region, name);
```

Comment:
Pre-existing, but this should be `tcx.hir().span_if_local(def_id).unwrap_or(DUMMY_SP)` (where the def_id comes from the other field of `BrNamed`). The other use of `get_named_span` above should also be similarly converted.

Reply:
@matthewjasper `get_named_span` is a method in the `region_name` submodule, and these are its only two uses. Are you suggesting that I get rid of it altogether?

Reply:
Yes